Samples/DynamicDependencies/DirectX/D3D9ExSample.vcxproj

```diff
@@ -27,20 +35,36 @@
     <PlatformToolset>v142</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
+    <ConfigurationType>Application</ConfigurationType>
+    <PlatformToolset>v142</PlatformToolset>
```

Comment:
We'll likely have problems with this when building with VS2022. Maybe trying creating a branch from mine instead of from main 
https://github.com/microsoft/WindowsAppSDK-Samples/tree/user/ertorres/build-samples-vs2022